docs/Build-for-RPi2.md

```diff
@@ -0,0 +1,62 @@
+## Build IoT.js with RPi2
+
+### Prerequisite
+* #### Linux
```

Comment:
This is a very strange style (heading in a list). Just use a plain simple heading. And then the text body will need no full-tab/8-chars/4-chars indentation.